runelite-client/src/main/java/net/runelite/client/plugins/chathistory/ChatHistoryPlugin.java

```diff
@@ -108,6 +159,14 @@ public void onChatMessage(ChatMessage chatMessage)
 				messageQueue.offer(queuedMessage);
 			}
 		}
+
+		if (CYCLE_TYPES.contains(chatMessage.getType()))
```

Comment:
This was just a switch in the other PR which seems like it is probably cleaner?

Reply:
switched to a switch statement